Mena_4pm_ProjectPortfolio_Team12.c

```diff
@@ -263,9 +263,9 @@ int CheckSpot(int row, int col, int board[][7]) {
 }
 int CheckWinner(int board[][7], int winningSpace[]) {
```

Comment:
yep it seems to run better now. the only issue I found was it said the function reached the end of a non void function with no return so instead of return 1 or return 0 in the loops, I made a new variable x in the function and set x = 1 or x = 0. I put return x at the end of the function and the error was cleared up.